src/workerinfo.js

```diff
@@ -183,13 +182,14 @@ class WorkerInfo {
       return;
     }
 
-    const recentTasks = worker.recentTasks.clone();
+    const recentTasks = [...worker.recentTasks];
 
-    tasks.forEach((task, index) => {
-      const taskId = tasks[index].status.taskId;
+    tasks.forEach((task) => {
+      const {taskId, runs} = task.status;
+      const runId = runs.length;
```

Comment:
This will start at 1 for runId 0, won't it?

Reply:
(I'm thinking yes given all the runId: 1 in the tests..)

Reply:
Good catch!